### PR TITLE
Doc tweaks

### PR DESCRIPTION
**Pull request by jpereira**

### CODE REVIEW

File: doc/howto/.gitignore [hunk 1]
```diff
@@ -1 +1 @@
-!*.md
+*.adoc
```
Comment (alandekok) [inline doc/howto/.gitignore]: All of the future howtos are going to be adoc files in that directory.  So we shouldn't ignore adoc files here.

File: doc/.gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 *.html
+*.adoc
```
Comment (alandekok) [inline doc/.gitignore]: We should generally *allow* adoc files, and only ignore the auto-generated ones.  Why is there a blanket "ignore adoc" files in the "doc" directory?

Comment (jpereira) [reply doc/.gitignore]: @alandekok I thought that we should hide the dynamic *.adoc.

```
[jpereira@sugarloaf freeradius-server.git]$
[jpereira@sugarloaf freeradius-server.git]$ git st
On branch v4/doc-tweaks2
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	doc/README.adoc
	doc/directories.adoc
	doc/howto/INSTALL.adoc
	doc/howto/README.adoc
	doc/howto/ascend.adoc
	doc/howto/bay.adoc
	doc/howto/cisco.adoc
	doc/howto/modules/RADIUS-LDAP-eDirectory.adoc
	doc/howto/modules/eap.adoc
	doc/howto/modules/expiration.adoc
	doc/howto/modules/krb5.adoc
	doc/howto/modules/ldap_howto.adoc
	doc/howto/modules/mschap.adoc
	doc/howto/modules/pam.adoc
	doc/howto/modules/passwd.adoc
	doc/howto/modules/python.adoc
	doc/howto/modules/soh.adoc
	doc/howto/modules/sql.adoc
	doc/howto/modules/sqlcounter.adoc
	doc/howto/modules/sqlippool.adoc
	doc/howto/performance-testing.adoc
	doc/howto/proxim.adoc
	doc/howto/supervise-radiusd.adoc
	doc/howto/tuning_guide.adoc
	doc/schemas/logstash/README.adoc

nothing added to commit but untracked files present (use "git add" to track)
[jpereira@sugarloaf freeradius-server.git]$
```

File: doc/howto/.gitignore (hunk 1 above)
Comment (jpereira) [reply doc/howto/.gitignore]: I updated the pr. now only the doc updates.

**Comment by arr2036** [comment]
@alandekok agreed to unify the format as adoc instead of a mixture of .md and .adoc, so you can probably just commit the adoc versions and remove the dynamic conversion

**Comment by jpereira** [comment]
@alandekok done.

File: doc/.gitignore (hunk 2 above)
Comment (alandekok) [reply doc/.gitignore]: Adding `*.adoc` to the `.gitignore` file *also* hides new `adoc` files that we create manually.

**Pull request merged**